kidiff_linux.py

```diff
@@ -703,7 +703,7 @@ def getGitDiff(diff1, diff2, prjctName, prjctPath):
     artifact2 = diff2[:6]
 
     findDiff = 'cd ' + _escape_string(prjctPath) + ' && ' + gitProg + ' diff --name-only ' + \
-        artifact1 + ' ' + artifact2 + ' | ' + grepProg + ' .kicad_pcb'
+        artifact1 + ' ' + artifact2 + ' . | ' + grepProg + ' .kicad_pcb'
```

Comment:
This looks fine to me, works as described. Just limits the `git diff` to the current directory.

I see no issues.